### ISSUE TITLE
Cannot create alerting policy

### ISSUE THREAD

**Issue by sethvargo** [google/exposure-notifications-verification-server#659]
```
Error: Error creating AlertPolicy: googleapi: Error 400: Field aggregation.perSeriesAligner had an invalid value of "ALIGN_RATE": The aligner cannot be applied to metrics with kind DELTA and value type DISTRIBUTION.
```

/cc @femnad @icco

**Comment by femnad** [comment]
Addressed by #667

**Comment by sethvargo** [comment]
Thanks!
